news/2020-04-04-beatmap-spotlights-winter-2020.md

```diff
@@ -0,0 +1,239 @@
+
+*written by [Yumeno Himiko](https://osu.ppy.sh/users/1806962)*
+
+[![](/wiki/shared/news/2020-04-04-beatmap-spotlights-winter-2020/catch/loving.jpg)](https://osu.ppy.sh/beatmapsets/1080025)
+
+'How on earth did eroge get into spotlights?' may be a question that pops up when first encountering this set. Despite the general stigma against the genre, [Luminate](https://osu.ppy.sh/users/4778689)'s "Loving Trip" does a great job of following the song.
+
+Slower parts of the song keep things interesting with varied rhythm and patterns, while higher density sections have liberal use of antiflow that contrasts well. This is one of the few sets that I've genuinely had fun playing this year, and I hope you give it a shot as well.
+
+*written by [Sinnoh](https://osu.ppy.sh/users/4236057)*
+
+[![](/wiki/shared/news/2020-04-04-beatmap-spotlights-winter-2020/catch/lights.jpg)](https://osu.ppy.sh/beatmapsets/1044462)
+
+[Kukkai](https://osu.ppy.sh/users/7811952)'s "Lights of Muse" has a certain feeling behind it that is fairly uncommon these days, reminiscent of something that would have easily had a play count in the hundred thousand if it was ranked a few years ago. While the use of patterns and rhythm is fairly simple, the whole set is at a consistent level of fun that is hard to come by.
+
+*written by [Sinnoh](https://osu.ppy.sh/users/4236057)*
+
+[![](/wiki/shared/news/2020-04-04-beatmap-spotlights-winter-2020/catch/sweet.jpg)](https://osu.ppy.sh/beatmapsets/738169)
+
+[Sorcerer](https://osu.ppy.sh/users/1913190) and [ZiRoX](https://osu.ppy.sh/users/200768) have something sweet to offer in this season's Spotlights!
+
+Starting from the bottom we have Cup, Salad and Platter all made by Sorcerer, and they lead the player through the learning stages of the streams and jumps. Platter introduces Hyperjumps and even more advanced stream patterns.
+
+Next up we have two Rain to choose from, one from the mapper of the previous three and one guest difficulty from ZiRoX. Dispite their similar star ratings there is a clear difference in them. ZiRoX decided to use less Hyperjump during the stream parts but instead use stair patterns, which are not necessarily easier to handle though. Meanwhile Sorcerer decided to step up from Platter and make his Hyperjump streams even more deadly for the player with more experience.
+
+Next up we have two Rain to choose from, one from the mapper of the previous three and one guest difficulty from ZiRoX. Despite their similar star ratings, there is a clear difference in them. ZiRoX uses less Hyperjump during the stream parts but went for stair patterns and some quick 1/4 slider jump pattern later on. Meanwhile, Sorcerer decided to step up from Platter and make his Hyperjump streams even more deadly for the player with more experience.
```

Comment:
Duplicated similar paragraphs. Picked second as it seems more developed, and I just did a plural fix in this suggestion.
```suggestion
Next up we have two Rain to choose from, one from the mapper of the previous three and one guest difficulty from ZiRoX. Despite their similar star ratings, there is a clear difference in them. ZiRoX uses less Hyperjump during the stream parts but went for stair patterns and some quick 1/4 slider jump patterns later on. Meanwhile, Sorcerer decided to step up from Platter and make his Hyperjump streams even more deadly for the player with more experience.
```